homeassistant/components/mqtt/device_tracker/__init__.py

```diff
@@ -6,23 +6,15 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
-from ..const import MQTT_DATA_DEVICE_TRACKER_LEGACY
 from ..mixins import warn_for_legacy_schema
-from .schema_discovery import PLATFORM_SCHEMA_MODERN  # noqa: F401
-from .schema_discovery import async_setup_entry_from_discovery
-from .schema_yaml import (
-    PLATFORM_SCHEMA_YAML,
-    MQTTLegacyDeviceTrackerData,
-    async_setup_scanner_from_yaml,
+from .schema_discovery import (  # noqa: F401
```

Comment:
In a follow-up PR, let's move contents of `homeassistant/components/mqtt/device_tracker/schema_discovery.py` to here

Reply:
Already thought about that, but for readability and traceability I left that part out here.